backend/FwLite/LcmCrdt/Changes/CreateSenseChange.cs

```diff
@@ -29,7 +29,7 @@ private CreateSenseChange(Guid entityId, Guid entryId) : base(entityId)
     public double Order { get; set; }
     public MultiString? Definition { get; set; }
     public MultiString? Gloss { get; set; }
-    public string? PartOfSpeech { get; set; }
+    public PartOfSpeech? PartOfSpeech { get; set; }
```

Comment:
you can actually just remove this whole property, since the Sense will just store the PartOfSpeech id we don't need anything else in the change.